src/components/Card.js

```diff
@@ -1,21 +1,34 @@
 export default class Card {
-  constructor({ name, link }, cardSelector, handleCardClick) {
+  constructor(
+    { name, link, _id, isLiked },
+    cardSelector,
+    handleCardClick,
+    handleLikeClick,
+    handleDeleteClick,
+    handleDeleteCard
+  ) {
     this._name = name;
     this._link = link;
+    this.id = _id;
+    this.isLiked = isLiked;
     this._cardSelector = cardSelector;
     this._handleCardClick = handleCardClick;
+    this._handleLikeClick = handleLikeClick;
+    this._handleDeleteClick = handleDeleteClick;
+    this._handleDeleteCard = handleDeleteCard;
   }
 
   _setEventListeners() {
     this._cardElement
       .querySelector(".card__like-button")
```

Comment:
You've found the like button in line 75. Please, use `this._likeButton` here

`Class fields` (the so-called **class variables** with `this`) are accessible inside any class method. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.